manifests/unit_file.pp

```diff
@@ -27,12 +27,26 @@
 #
 #   * Mutually exclusive with both ``$source`` and ``$content``
 #
+# @attr manage_service
```

Comment:
As @jcharaoui said, a unit can be anything beside a service thus I find the names of these parameters to be confusing.

Maybe `$ensure_service` could be computed from `$ensure` and `$manage_service` (that would be renamed `$manage_unit` or just `$manage` idk) and `$enable_service` could then be renamed `$enable`.